src/client/client.ts

```diff
@@ -1,10 +1,13 @@
+import * as path from 'path';
 import { ChildProcess, fork, ForkOptions } from 'child_process';
 import { EventEmitter } from 'events';
 
+const tscWatchLibPath = require.resolve(path.resolve(__dirname, '..', 'lib', 'tsc-watch'));
```

Comment:
why are you using relative path instead of `resolve`? the `resolve` handles all the library searching mechanism...

Reply:
@gilamran I can't use the lib in my project since the client [move from `lib/` to `client/`](https://github.com/gilamran/tsc-watch/commit/a78852b724c092e3e448a222c5a737277a893c2b#diff-ed2b939cef6b7455068ab86c443272afeb42f47a0264b82ec98fbd19d774e4d1) and `require.resolve` fails to file `tsc-watch` 

But require.resolve is now useless, or do you see better? 994e40741ef4f4da08af943a3962666457035ab6

Reply:
ok got you.
how about just `path.join(__dirname, '..', 'lib', 'tsc-watch')`?

Reply:
@gilamran Sure! I replaced my previous commit 994e40741ef4f4da08af943a3962666457035ab6 by this one ce7ac7846aa5b8cfb6080035b865ada5e428d557